bot/discord/manager.go

```diff
@@ -147,10 +149,12 @@ func getBloopyChanHandler(ds *services.DiscordService, msgSendChan *chan *models
 	return handlers.NewMessageChanBlooper(ds, providers.GetInspiroService(), &createCh, &reactACh, &reactRCh, msgSendChan)
 }
 
+// IsReady returns true if the Discord service is ready.
 func (d *DiscordManager) IsReady() bool {
 	return d.discordSvc.GetDataReady()
 }
 
```

Comment:
More specifically it returns the discord service instance stored in this instance of the manager.